variables.ini

```diff
@@ -51,4 +51,5 @@ UBOOT_ITB_URL=variable-value
 KERNEL_FIT_URL=variable-value
 KERNEL_ZIMAGE_URL=variable-value
 NFSROOTFS_DEV_URL=variable-value
+auto_login_prompt=prompt-value
```

Comment:
I would like to keep a convention when all externally set variables are in upper case. Variables used internally are lower case.

Reply:
yes